buildscript.py

```diff
@@ -0,0 +1,34 @@
+import os
```

Comment:
Let's move the script to `bin/`, rename to just `bundle-extensions`, make it executable and add a shebang line as the first line:

```
#!/usr/bin/env python
```

That way the script is self-contained and can be called from npm.